schemas/styles/csl.rnc

```diff
@@ -893,6 +897,48 @@ div {
         "partial-first"
     }?
   
+  ## Options affecting rendering of title variables, for cs:style, cs:citation and cs:bibliography
+  title-inheritable-options =
+    
+    ## Inheritable title option, specify the delimiter between "title-main" and "title-sub".
```

Comment:
Is there a (perhaps locale-specific) default?

Reply:
An empty delimiter means that no normalization of the delimiters occurs